src/index.html

```diff
@@ -10,8 +10,175 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+  <body class="row">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="form"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="Surename">Surename:
+          </label>
+          <input type="text"
+            id="Surename"
+            name="Surname" multiple
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="Name">Name:
+          </label>
+          <input type="text"
+            id="Name"
+            name="Name"
+            autocomplete="off"
+          >
+        </div>
```

Comment:
Apply these recommendation to other similar cases as well:
1) it would be more readable to wrap input with label and avoid using id/for
2) check code styling [recommendations](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) such as points №2-4
3) div wrapper is redundant, you can apply styles to a label

```suggestion
        <label class="form-field">
          Name:
          <input 
            type="text"
            name="name"
            autocomplete="off"
          >
        </label>
```